cdk/dcm_floodlight/transform.js

```diff
@@ -91,7 +91,17 @@ function trackPostMapper(input, mappedPayload, rudderContext) {
     ...mappedPayload
   };
 
-  if (!salesTag) {
+  if (salesTag) {
```

Comment:
Here if the `products` array is present then override the `mappedPayload.qty` with  quantity calculated from number of products